src/main/java/drake/Ui.java

```diff
@@ -0,0 +1,125 @@
+package drake;
+import java.util.ArrayList;
+
+import drake.task.Task;
+import drake.task.TaskList;
+
+/**
+ * Handles UI operations for the chatbot.
+ */
+public class Ui {
+
+    /**
+     * Returns a welcome message to the user.
+     */
+    public static String showWelcome() {
+        return "____________________________________________________________\n" + " What's up everyone. I'm Drake.\n"
+                + " How can I help?\n"
+                + "____________________________________________________________";
+    }
+
+    /**
+     * Returns a goodbye message to the user.
+     */
+    public String showGoodbye() {
+        return "____________________________________________________________\n"
+                + " See you later, alligator! \n"
+                + "____________________________________________________________";
+    }
+
+    /**
+     * Returns an error message.
+     *
+     * @param message The error message to be returned.
+     */
+    public String showError(String message) {
+        return "____________________________________________________________\n"
+                + message + "\n"
+                + "____________________________________________________________";
+    }
+
+    /**
+     * Returns the list of tasks as a string.
+     *
+     * @param tasks The TaskList containing the tasks to display.
+     */
+    public String showTaskList(TaskList tasks) {
+        StringBuilder sb = new StringBuilder();
+        sb.append("____________________________________________________________\n");
```

Comment:
Try abstracting out the divider line code that is reused in multiple functions like showAddTask, showTaskList etc to a seperate divider function. this make the code clumsy

Reply:
```suggestion
     public void divider() {
        System.out.println("____________________________________________________________\n");
        
        }
```